pages/contact-us.md

```diff
@@ -54,6 +54,11 @@ follow discussions as they happen.
 
 Drop by and chat about Accumulo at the [#accumulo][accumulo-channel] channel on the [ASF Slack workspace][asf-slack]. Join using this [invite link][slack-invite]. If you have trouble joining, email `dev@accumulo.apache.org` for an invite.
 
+## Confluence
+
+The [Accumulo Confluence Space][confluence] provides a collaboration space for design documents.  If you would like write 
+access, please send a request to `dev@accumulo.apache.org` with your Confluence username.
+
```

Comment:
I don't think the contact page is the appropriate place for this. I think it would be better to mention the use of Confluence on a page like https://accumulo.apache.org/how-to-contribute/, maybe under the "Helpful Links" section, or maybe in its own section referring to developer design documentation. It should link back to the contact-us page instead of directly linking the mailing list name (just to keep it listed in one place, in case it changes).

The how to contribute page is actually a bit sparse. Awhile back, somebody on the ComDev mailing list suggested standardizing on projects creating a `/contribute` entry point to collect project-specific information on contributing. I think our "how to contribute" page kind of does that, but could perhaps use a bit of a refresher, including adding this info about Confluence.

Reply:
Added a reference on how-to-contribute and a link back to the contact-us.

Being that the Confluence space is a communications channel and one way to interact with the Accumulo community, the contact-us page seems a logical place to look if someone is searching for a way to interact with the community.

The intention of this PR was to raise the visibility of the Confluence space. The refresh of the `/contribute` page can be done as a separate effort.

Reply:
addressed in 110837d0